core/base/PeriodicImplicitTriangulation/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+ttk_add_base_library(periodicImplicitTriangulation
```

Comment:
to comply with ttk's naming convention, please rename (git mv)
- core/base/PeriodicImplicitTriangulation/ into  core/base/periodicImplicitTriangulation/
thanks for changing that in the involved cmake files.

Reply:
Done.